codebase/analytics-gemini/client.go

```diff
@@ -0,0 +1,217 @@
+package gemini
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+
+	"github.com/fabric8-services/fabric8-wit/client"
+	"github.com/fabric8-services/fabric8-wit/errors"
+	"github.com/fabric8-services/fabric8-wit/goasupport"
+	"github.com/fabric8-services/fabric8-wit/log"
+	"github.com/fabric8-services/fabric8-wit/rest"
+
+	goaclient "github.com/goadesign/goa/client"
+	"github.com/goadesign/goa/middleware"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+	errs "github.com/pkg/errors"
+)
+
+type response struct {
+	Summary string `json:"summary"`
+	Error   string `json:"error"`
+}
+
+type request struct {
+	gitURL   string   `json:"git-url"`
+	emailIDs []string `json:"email-ids"`
+}
+
+func NewScanRepoRequest(repoURL, email string) *request {
+	return &request{
+		gitURL:   repoURL,
+		emailIDs: []string{email},
```

Comment:
What are these emailIds?

Reply:
they ask for user email id, which they might use for some other analytics purpose.

Reply:
But we have none to send?

Reply:
Since not used, I would preferre if we removed it from our codebase

Reply:
removed it all!